### PR TITLE
https://maplibre.org/about/

### PR DESCRIPTION
**Pull request by duje**
Add About Duje

### CODE REVIEW

File: content/about/duje/_index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+.'--'.  _.'    {`'-;_ .-...'
+      '-:_      )  / `' '=.
+        ) >     {_/     /.)..
+        |/               `..; .'
+</div>
+
+
+End? No, the journey doesn't end here. MapLibre is another path. One that we
+all must take. On 2020-12-08 22:42:56 UTC MapLibre restarted mapping with its
+mission to provide freedom and independence. Eventually, this is what everybody
```
Comment (lseelenbinder) [inline content/about/duje/_index.md:28]:
```suggestion
mission to provide freedom and independence. In the end, this is what everybody
```

File: content/about/duje/_index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+is seeking, from a little project to enterprise endeavours. I am Duje and I am
```
Comment (lseelenbinder) [inline content/about/duje/_index.md:29]:
```suggestion
is seeking, from a little project to enterprise endeavours. I am Duje, and I am
```

File: content/about/duje/_index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+assisting as a Coordinator to help maintain what the community has built so
+carefully which means to me:
```
Comment (lseelenbinder) [inline content/about/duje/_index.md:31]:
```suggestion
carefully. For me, this means:
```

File: content/about/duje/_index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+- Assure MapLibre's fundraising and a healthy Sponsorship Program to keep
+  MapLibre on the map
+- Listen to and discuss with the vivid community which corner to turn next
```
Comment (lseelenbinder) [inline content/about/duje/_index.md:35]:
```suggestion
- Listen to and discuss with the vibrant community which corner to turn next
```

English nitpick.

"vivid" usually is used with colors; "vibrant" is used with colors and people. :)

File: content/about/duje/_index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- Make MapLibre seen and accessible to everyone
+
+To sidewalk talks where everyone feels comfortable in the MapLibre world.
```
Comment (lseelenbinder) [inline content/about/duje/_index.md:38]:
```suggestion
Here's to sidewalk talks where everyone feels comfortable in the MapLibre world.
```

It took me a couple reads to realize what you meant. :smile:

File: content/about/duje/_index.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+Gladly find me in my alley: <span><a
```
Comment (lseelenbinder) [inline content/about/duje/_index.md:40]: I'd rephrase this, as well. Something like "You can always find me on Slack". This phrase is a bit awkward in English.

Comment (wipfli) [reply content/about/duje/_index.md:40]: @duje I will leave this one up to you. Let's merge it like this and if you want to update it, feel free to submit a follow-up pull request.

**Review by wipfli — APPROVED** [review]
Thanks @duje and welcome to MapLibre!

**Comment by wipfli** [comment]
Can you also add yourself here, @duje ?
image

File: content/about/duje/_index.md (hunk 1 above)
Comment (duje) [reply content/about/duje/_index.md]: https://dictionary.cambridge.org/dictionary/english/eventually 🤔

"In the end" sounds final as in at the end of a period, whereas "eventually" feels more like "some day (in the future)" and less opinionated or like a statement.

**Comment by duje** [comment]
@wipfli Will redo all in a new branch, or pull request.

File: content/about/duje/_index.md (hunk 6 above)
Comment (duje) [reply content/about/duje/_index.md:39]: Let it be a nice alley that one would find in Dubrovnik, Lisbon or Zurich 🙂
